### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by SalahAdDin**
Fix references for `Odoo` 11.0.

### CODE REVIEW

File: currency_rate_update/README.rst [hunk 1]
```diff
@@ -14,11 +14,14 @@ The reference rates are based on the regular daily query procedure between centr
 Configuration
 =============
 
-To configure the module, go to *Accounting > Configuration > Multi-currencies > Rate Auto-download* and create one or several services to download rates from the Internet.
+To configure the module:
 
-Then, go to the page *Accounting > Configuration > Settings* and, in the section *Multi Currencies*, make sure that the option *Automatic Currency Rates Download* is enabled.
+1. Go to *Invoicing > Configuration > Rate Auto-download* and create one or several services to download rates from the Internet.
```
Comment (elicoidal) [inline currency_rate_update/README.rst]: better use ´#. ´ for enumerated list (and in this case, no need empty line in the list)

File: currency_rate_update/README.rst [hunk 2]
```diff
@@ -14,11 +14,14 @@ The reference rates are based on the regular daily query procedure between centr
...
+
+2. Go to the page *Invoicing > Configuration > Settings* and, in the section *Multi Currencies*, make sure that the option *Automatic Currency Rates Download* is enabled.
+
+3. In developper mode, in the menu *Settings > Technical > Scheduled Actions*, make sure that the action *Currency Rate Update* is active. If you want to run it immediately, use the button *Run Manually*.
 
-In developper mode, in the menu *Settings > Technical > Scheduled Actions*, make sure that the action *Currency Rate Update* is active. If you want to run it immediately, use the button *Run Manually*.
 
 This module is compatible with OCA module 'currency_rate_inverted' also found in OCA/currency repository, that allows to maintain exchange rates in inverted format, helping to resolve rounding issues.
```
Comment (elicoidal) [inline currency_rate_update/README.rst:28]: ~Actually the module currency_rate_inverted is not available for v11: I would remove the sentence~

**Review by elicoidal — APPROVED** [review]
LGTM

**Comment by SalahAdDin** [comment]
I tried using a list, but it didn't work.

And, that module is available for `11.0`, i could install it.

**Comment by elicoidal** [comment]
> And, that module is available for 11.0, i could install it.

I did not find it last time: if you rechecked then that is OK.

**Comment by elicoidal** [comment]
Cannot access diff for Github error: will retry later

File: currency_rate_update/README.rst (hunk 1 above)
Comment (elicoidal) [reply currency_rate_update/README.rst]: Something similar to:
```
#. Go to *Invoicing > Configuration > Rate Auto-download*
#. Create one or several services to download rates from the Internet.
#. Go to the page *Invoicing > Configuration > Settings* 
#. In the section *Multi Currencies*, make sure that the option *Automatic Currency Rates Download* is enabled.
#. Switch to developper mode, 
#. in the menu *Settings > Technical > Scheduled Actions*, make sure that the action *Currency Rate Update* is active. 
#. If you want to run it immediately, use the button *Run Manually*.

**Comment by SalahAdDin** [comment]
@elicoidal https://github.com/OCA/currency/tree/11.0/currency_rate_inverted

**Comment by elicoidal** [comment]
@SalahAdDin As said: I believe you :smile:

**Comment by SalahAdDin** [comment]
@elicoidal I don't understand you :')

**Comment by levin222** [comment]
@SalahAdDin  My Settings are the same as yours, but it still fail . Is that OK for U right now ?

**Comment by SalahAdDin** [comment]
@levin222 Yes, you must to put your currency as default in your company and in your _**invoice settings**_, after that, in invoicing, you must to create a new **_Rate_**..., after you create it immediately you can update your currency's rates.

**Comment by LoisRForgeFlow** [comment]
@SalahAdDin Can you squash all commits? then we can merge.

**Comment by SalahAdDin** [comment]
Next week i can see it, thanks.

**Pull request closed without merging**